tests/sqlparser_hive.rs

```diff
@@ -29,6 +29,12 @@ fn parse_table_create() {
     hive().verified_stmt(iof);
 }
 
+#[test]
+fn parse_describe() {
```

Comment:
Given this was a change to the generic Dialect (the default impl) can you also please add a test in `parser_generic.rs` as well?

Reply:
Yep, I think that makes sense. Thank you! 3rd kid in as many years born Monday (first daughter!), will get to it in a few days I think.

Reply:
> 3rd kid in as many years born Monday (first daughter!

Congratulations and good luck!

Reply:
>can you also please add a test in parser_generic.rs as well?

I've added a generic test in the Hive test. parser_common was mostly about "all dialects", this change is just hive and generic. There are other Hive tests testing generic such that I factored out the dialect value creation into a function.

I think the remaining thing here is lifting the backtick out of the trait default implementation out to Hive and Generic's trait impls. What do you think?